firebase-inappmessaging/src/test/java/com/google/firebase/inappmessaging/FirebaseInAppMessagingTest.java

```diff
@@ -149,7 +150,11 @@ public Builder toBuilder() {
   @Mock private DisplayCallbacksFactory displayCallbacksFactory;
   @Mock private FirebaseInAppMessagingDisplayCallbacks displayCallbacks;
   @Mock private ProgramaticContextualTriggers programaticContextualTriggers;
-  @Mock DeveloperListenerManager developerListenerManager = new DeveloperListenerManager();
+
+  @Mock
+  DeveloperListenerManager developerListenerManager =
+      new DeveloperListenerManager(Executors.newSingleThreadExecutor());
```

Comment:
nit: pls use `TestOnlyExecutors` to inject the right executor for tests as that would ensure no unintended IO/blocking is done and fail tests otherwise.

to use them add a dep `testImplementation project(":integ-testing")`

Reply:
Done